string interested_object
---
bool is_done